tools/replaceColumn/replaceColumn.xml

```diff
@@ -0,0 +1,109 @@
+<tool id="replace_column_with_key_value_file" name="Replace column" version="0.1">
+    <description>by values which are defined in a convert file</description>
+    <command>
+        <![CDATA[
+        python '$replaceColumnScript'
+        ]]>
+    </command>
+    <configfiles>
+        <configfile name="replaceColumnScript">
+            <![CDATA[
+import sys
+
+replace_file = '$replace_information'
+original_file = '$original_file'
+column = int("$column_replace") - 1
+ignore_start_lines = int("$skip_lines")
+delimiter_local = "\t" if str("$delimiter") == "" else str("$delimiter")
+
+## read conversion information to index 
+conversion = {}
+
+with open(replace_file, 'r') as conversion_file:
+    for line in conversion_file:
+        conv_key_value = line.strip().split()
+        if len(conv_key_value) == 2:
+            conversion[conv_key_value[0]] = conv_key_value[1]
+        else:
+            conversion[conv_key_value[0]] = ''
```

Comment:
Does this mean that the conversion will create a column with an empty value? So the chromosome column is empty after conversion?